sunpy/visualization/colormaps/color_tables.py

```diff
@@ -81,7 +82,7 @@ def paden(lst, len_, pad=0):
 }
 
 
-def aia_color_table(wavelength):
+def aia_color_table(wavelength: u.angstrom):
```

Comment:
```suggestion
@u.quantity_input
def aia_color_table(wavelength: u.angstrom):
```

As we require from: https://docs.sunpy.org/en/latest/dev_guide/units_quantities.html

Reply:
okay i got it i guess..thanks:-)

Reply:
as i don't import astropy.units as u first it errors as  'u' is not recognised..

Reply:
Well the import ideally should be at the top of the file. So it would work.